### PR TITLE
chore(config): don't symlink hexo-theme-doc

### LINKED ISSUE #112: More convienent way of making Hexo use the theme from node_modules?
So, I was wondering, why do we have to make a symlink to install the theme? It's in order to get Hexo to use the theme from node_modules, right? So that the theme can be updated with npm, yarn or whatever

Well, wouldn't the same thing be achieved by setting the theme option to that folder directly?

Like this:
```yaml
theme: ../node_modules/hexo-theme-doc
```

Because I did test it and it seems to be working, so I was wondering if you did consider this, maybe there are some "side effects" but as far as I can tell, it seems to be working, so I thought I would ask about it

### PR DESCRIPTION
**Pull request by rbarilani**
Don't symlink `node_modules/hexo-theme-doc` as `themes/doc`.
Reference a relative path in the configuration file as `theme`
as suggested here: #112.

##### TODO for #112 

- Update hexo-theme-doc-seed

### CODE REVIEW

**Comment by rbarilani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Comment by bhaskarmelkani** [comment]
👍

File: _config.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ timezone: UTC
 
 # URL
 ## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
-url: https://zalando-incubator.gtihub.io/
+url: https://zalando-incubator.gihub.io/
```
Comment (bhaskarmelkani) [inline _config.yml]: typo `https://zalando-incubator.github.io/`

Comment (rbarilani) [reply _config.yml]: @bhaskarmelkani fixed

**Comment by rbarilani** [comment]
👍

**Comment by bhaskarmelkani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Pull request merged**